index.html

```diff
@@ -64,5 +66,182 @@ <h1>I’m MERCY <br />Glad to see <br />you!</h1>
         </ul>
       </div>
     </section>
+    <!--Start of detailed section-->
+    <section class="projects-section" >
+      <!--Card1-->
+      <div class="cards">
+        <img src="/assets/images/project1.png" alt="project1" />
+      </div>
+      <div class="cards-details">
+        <div>
+        <h2 class="title">Tonic</h2>
+        <div class="highlights">
+          <p class="highlight-text">Canopy</p>
+            <img class="counter" src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">Back End Dev</p>
+            <img src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">2015</p>
+        </div>
+        </div>
+        <p class="cards-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="grid-skills">
+          <div class="item1">html</div>
+          <div class="item2">css</div>
+          <div class="item3">javascript</div>
+        </div>
+        <button id="popup">see project</button>
+      </div>
+    </section>
```

Comment:
Well done on the `section` tag here 😉 but the requirement says that the container of the **Works** section should be a `section` tag. You indeed used a `section` tag but to wrap each work section, you're supposed to have a `section` tag wrapping all the other sections that you have for each work. This is what your structure looks like 👇🏽 
![Screenshot 2023-02-17 at 21 15 20](https://user-images.githubusercontent.com/45739963/219826495-ffe9fcb3-9b20-484c-928e-2bd5d3d201d0.png)

Also because of that, you're not fulfilling another requirement which is that the main container of the projects has a `display:grid`. This will help also when it's time to make your page responsive. 💯